lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -98,6 +98,7 @@ export default function OAuth2RedirectErrorApp({ location = window.location }) {
           <p>
             For more information, please have your Blackboard admin read:{' '}
             <Link
+              classes="inline"
```

Comment:
I just spotted this while I was in here. `Link` elements are `inline-block` by default, which works for most links because the text is short and fits on the same line. This allows this longer link to wrap, and start on the same line.